bridges/TelegramBridge.php

```diff
@@ -132,6 +132,14 @@ private function processContent($messageDiv) {
 				$messageDiv->find('div.tgme_widget_message_text.js-message_text', 0)->plaintext
 			);
 		}
+		
+		if ($messageDiv->find('div.tgme_widget_message_document', 0)) {
+			$message .= "Attachments:";
```

Comment:
Single quotes should be used.
```suggestion
			$message .= 'Attachments:';
```